app/src/main/java/com/abisoft/todocompose/ui/viewModel/addTaskViewModel/AddTaskViewModel.kt

```diff
@@ -0,0 +1,71 @@
+package com.abisoft.todocompose.ui.viewModel.addTaskViewModel
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.abisoft.todocompose.TodoItemsRepository
+import com.abisoft.todocompose.model.ImportanceNetwork
+import com.abisoft.todocompose.model.TodoItemNetwork
+import com.abisoft.todocompose.model.TodoItemPost
+import com.example.myapplication.okhttp_client.TodoDto
+import kotlinx.coroutines.launch
+import java.util.UUID
+
+
+class AddTaskViewModel(private val repository: TodoItemsRepository) : ViewModel() {
+
+        private val _todoList = MutableLiveData<List<TodoItemNetwork>>()
```

Comment:
1. For our project you should use MutableStateFlow in ViewModel instead. Also you shouldn't mix up these 2 classes in same project for consistency. You can read about differences in Internet (mostly it is lifecycle control). Usually in new projects we prefer Flow 'cause it haven't Android dependency, coroutine-friendly, more flexible & testable.
2. Tabs formatting is broken